images/runtime/2.7/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.12
+
+RUN apk update && apk upgrade \
+  && apk add ca-certificates \
+  && rm -rf /var/cache/apk/*
+
+RUN addgroup -g 1000 -S fn && adduser -S -u 1000 -G fn fn
```

Comment:
Why are we adding this new group here ?

Reply:
This isn't new, we add this group and user to all the FDK images.